packages/server/graphql/mutations/helpers/addAIGeneratedContentToThreads.ts

```diff
@@ -0,0 +1,60 @@
+import {AIExplainer} from '../../../../client/types/constEnums'
+import {PARABOL_AI_USER_ID} from '../../../../client/utils/constants'
+import getRethink from '../../../database/rethinkDriver'
+import Comment from '../../../database/types/Comment'
+import DiscussStage from '../../../database/types/DiscussStage'
+import {convertHtmlToTaskContent} from '../../../utils/draftjs/convertHtmlToTaskContent'
+import {DataLoaderWorker} from '../../graphql'
+
+const addAIGeneratedContentToThreads = async (
+  stages: DiscussStage[],
+  meetingId: string,
+  teamId: string,
+  dataLoader: DataLoaderWorker
+) => {
+  const [r, groups, team] = await Promise.all([
+    getRethink(),
+    dataLoader.get('retroReflectionGroupsByMeetingId').load(meetingId),
+    dataLoader.get('teams').loadNonNull(teamId)
+  ])
+  const {tier} = team
+  const commentPromises = stages.map(async (stage, idx) => {
+    const group = groups.find((group) => group.id === stage.reflectionGroupId)
+    if (!group?.summary && !group?.discussionPromptQuestion) return
```

Comment:
-1 If only one of those is not defined, we will still add both comments, one of them will just be undefined